src/mumble/Settings.cpp

```diff
@@ -512,7 +513,7 @@ Settings::Settings() {
 	bEnableUIAccess             = true;
 
 	for (int i = Log::firstMsgType; i <= Log::lastMsgType; ++i) {
-		qmMessages.insert(i, Settings::LogConsole | Settings::LogBalloon | Settings::LogTTS);
+	qmMessages.insert(i, Settings::LogConsole | Settings::LogBalloon | Settings::LogTTS | Settings::LogMessageLimit);
```

Comment:
```suggestion
		qmMessages.insert(i, Settings::LogConsole | Settings::LogBalloon | Settings::LogTTS | Settings::LogMessageLimit);
```
Doesn't this by default enable message limiting for all message types? I think in this case we should default to this feature being disabled.